src/components/PosterList.js

```diff
@@ -0,0 +1,46 @@
+import styled from 'styled-components';
+
+import Poster from './Poster';
+
+export default function PosterList({ list, listName }) {
+  return (
+    <>
+      <ListHeader>{listName}</ListHeader>
+      <StyledList>
```

Comment:
Add role="list" here for accessibility

Reply:
I had this in mind earlier and then forgot to add it, thank you very much